### PR TITLE
Add Strava provider

### PR DESCRIPTION
**Pull request by jamesholcomb**
#115

### CODE REVIEW

File: samples/Mvc.Client/project.json [hunk 1]
```diff
@@ -56,7 +57,7 @@
     "Microsoft.Extensions.Configuration.CommandLine": "1.0.0",
     "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0",
     "Microsoft.Extensions.Logging.Console": "1.0.0",
-    "Microsoft.Extensions.Logging.Debug": "1.0.0"
+   "Microsoft.Extensions.Logging.Debug": "1.0.0"
```
Comment (PinpointTownes) [inline samples/Mvc.Client/project.json]: Please revert.

File: src/AspNet.Security.OAuth.Strava/StravaAuthenticationHelper.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    /// Contains static methods that allow to extract user's information from a <see cref="JObject"/>
+    /// instance retrieved from Strava after a successful authentication process.
+    /// </summary>
+    public static class StravaAuthenticationHelper
+    {
+        /// <summary>
+        /// Gets the identifier corresponding to the authenticated user.
+        /// </summary>
+        public static string GetIdentifier([NotNull] JObject user) => user.Value<string>("id");
+        public static string GetUsername([NotNull] JObject user) => user.Value<string>("username");
```
Comment (PinpointTownes) [inline src/AspNet.Security.OAuth.Strava/StravaAuthenticationHelper.cs]: Please add the missing XML summaries.

**Comment by PinpointTownes** [comment]
Thanks for your contribution! :clap: 

Your PR looks fine. Please rebase your branch and add the missing XML documentation and I'll merge it ASAP :+1:

**Comment by jamesholcomb** [comment]
I'm a rebase newbie.  Since I did not create a local branch for this PR, will I still need to rebase and if so, how do you suggest I proceed?

Just want to ensure I get it right.

**Comment by PinpointTownes** [comment]
> I'm a rebase newbie. Since I did not create a local branch for this PR, will I still need to rebase and if so, how do you suggest I proceed?

It shouldn't different from rebasing a local branch:

```
git fetch aspnet-contrib
git rebase aspnet-contrib/dev
```
At this point, fix the conflicts (you can run `git status` to list them) and when you're done:

```
git add .
git rebase --continue
git push -f
```

**Comment by PinpointTownes** [comment]
Fixed, rebased and merged: https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/commit/8aa6bc913142f7c403df657485ad963c4751993d

@jamesholcomb thanks for your contribution! :+1:

**Pull request closed without merging**